regsync/regsync/cli.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import logging
+import os
+from pprint import pformat
+from regsync.deploy.registry import DEPLOY_REGISTRY
+
+
+def cli_init(args):
+    logger = logging.getLogger(__name__)
+    deploy_client = DEPLOY_REGISTRY[args.deploy_service]()
```

Comment:
It is a little confusing that these take the arg for the deploy type explicitly while run uses the env variable. I suggest having all of them read the env variable AND all allow manual cli overrides for consistency